These are needed only for static linking, but we build with --disable-static

--- origsrc/curl-7.33.0/curl-config.in	2013-09-08 17:11:15.000000000 -0500
+++ src/curl-7.33.0/curl-config.in	2013-11-13 20:43:10.231255800 -0600
@@ -147,7 +147,7 @@ while test $# -gt 0; do
         else
            CURLLIBDIR=""
         fi
-        if test "X@REQUIRE_LIB_DEPS@" = "Xyes"; then
+        if test "X@REQUIRE_LIB_DEPS@" = "Xnothanks"; then
           echo ${CURLLIBDIR}-lcurl @LIBCURL_LIBS@
         else
           echo ${CURLLIBDIR}-lcurl
--- origsrc/curl-7.33.0/libcurl.pc.in	2013-09-08 17:11:15.000000000 -0500
+++ src/curl-7.33.0/libcurl.pc.in	2013-11-13 20:42:48.656021800 -0600
@@ -35,5 +35,4 @@ URL: http://curl.haxx.se/
 Description: Library to transfer files with ftp, http, etc.
 Version: @CURLVERSION@
 Libs: -L${libdir} -lcurl
-Libs.private: @LIBCURL_LIBS@
 Cflags: -I${includedir} @CPPFLAG_CURL_STATICLIB@
